UnityProject/Assets/Scripts/Objects/Consoles/ArtifactConsole.cs

```diff
@@ -18,8 +19,7 @@ public class ArtifactConsole : NetworkBehaviour, ICheckedInteractable<HandApply>
 
 		[SyncVar] public ArtifactData inputData = new ArtifactData();
 
-		public delegate void StateChange();
-		public static event StateChange stateChange;
+		public Action StateChange;
```

Comment:
I think you still need to make it an event ``public event Action StateChange;``

Reply:
What's the benefit of making it an event? Been testing with client and host on a build and getting no errors with just the action.